install/installer/traf_config_setup

```diff
@@ -550,10 +562,10 @@ else
    exit -1
 fi
 
-hadoopVersion=$(curl -su $username:$password http://$URL/api/v1/clusters | grep version | grep -c CDH)
+hadoopVersion=$(curl -su admin:admin http://$URL/api/v1/clusters | grep version | grep -c CDH)
```

Comment:
Sorry, I guess the merge didn't happened as I thought it would. Let me fix and check in.